### DISCUSSION TITLE
Welcome to CrewAI Discussions!

### DISCUSSION THREAD

**Discussion by pythonbyte** [crewAIInc/crewAI#1077] [Announcements]
## 👋🏽 Welcome to CrewAI Discussions

We're using Discussions as a place to connect with other members of our community. 

We hope that you:

* Ask a question you're wondering about
* Share ideas
* Engage with other community members
* Welcome others and are open-minded. Remember that this is a community we build together 💪🏽 
* Help identify bugs, issues, or improvements.

Take a look at our [Documentation](https://docs.crewai.com/) and let us know how we can help improve your Agentic Workflow

**Comment by ctasoulas** [comment]
Hi,

Can I install crew ai via conda instead of pip?

Thanks

**Comment by ok71ers** [comment]
I've installed Python 3.11.9 on my Mac successfully.  However, when I tried to install Crewai followed the instructions, I got this error message:  "ERROR: Ignored the following versions that require a different python version: 0.0.2 Requires-Python >=3.10,<3.12; 0.0.3 Requires-Python >=3.10,<3.12; 0.1.0 Requires-Python >=3.9,<3.12; 0.1.1 Requires-Python >=3.9,<3.12; 0.1.10 Requires-Python >=3.9,<3.12; 0.1.11 Requires-Python >=3.9,<3.12; 0.1.12 Requires-Python >=3.9,<3.12; 0.1.13 Requires-Python >=3.9,<3.12; 0.1.14 Requires-Python >=3.9,<3.12; 0.1.15 Requires-Python >=3.9,<3.12; 0.1.16 Requires-Python >=3.9,<3.12; 0.1.17 Requires-Python >=3.9,<3.12; 0.1.18 Requires-Python >=3.9,<3.12; 0.1.19 Requires-Python >=3.9,<3.12; 0.1.19a1 Requires-Python >=3.9,<3.12; 0.1.19a2 Requires-Python >=3.9,<3.12; 0.1.19a3 Requires-Python >=3.9,<3.12; 0.1.19a4 Requires-Python >=3.9,<3.12; 0.1.19a5 Requires-Python >=3.9,<3.12; 0.1.2 Requires-Python >=3.9,<3.12; 0.1.20a1 Requires-Python >=3.9,<3.12; 0.1.21 Requires-Python >=3.9,<3.12; 0.1.22 Requires-Python >=3.9,<3.12; 0.1.23 Requires-Python >=3.9,<3.12; 0.1.24 Requires-Python >=3.9,<3.12; 0.1.25 Requires-Python >=3.9,<3.12; 0.1.26 Requires-Python >=3.9,<3.12; 0.1.27 Requires-Python >=3.9,<3.12; 0.1.28 Requires-Python >=3.9,<3.12; 0.1.3 Requires-Python >=3.9,<3.12; 0.1.30 Requires-Python >=3.9,<3.12; 0.1.31 Requires-Python >=3.9,<3.12; 0.1.33 Requires-Python >=3.9,<3.12; 0.1.34 Requires-Python >=3.9,<3.12; 0.1.35 Requires-Python >=3.9,<3.12; 0.1.36 Requires-Python >=3.9,<3.12; 0.1.37 Requires-Python >=3.9,<3.12; 0.1.38 Requires-Python >=3.9,<3.12; 0.1.39 Requires-Python >=3.9,<3.12; 0.1.4 Requires-Python >=3.9,<3.12; 0.1.40 Requires-Python >=3.9,<3.12; 0.1.41 Requires-Python >=3.9,<3.12; 0.1.41a0 Requires-Python >=3.9,<3.12; 0.1.41a1 Requires-Python >=3.9,<3.12; 0.1.42 Requires-Python >=3.9,<3.12; 0.1.43 Requires-Python >=3.9,<3.12; 0.1.44 Requires-Python >=3.9,<3.12; 0.1.45 Requires-Python >=3.9,<3.12; 0.1.45a0 Requires-Python >=3.9,<3.12; 0.1.46 Requires-Python >=3.9,<3.12; 0.1.47 Requires-Python >=3.9,<3.12; 0.1.48 Requires-Python >=3.9,<3.12; 0.1.49 Requires-Python >=3.9,<3.12; 0.1.5 Requires-Python >=3.9,<3.12; 0.1.50 Requires-Python >=3.9,<3.12; 0.1.51 Requires-Python >=3.9,<3.12; 0.1.52 Requires-Python >=3.9,<3.12; 0.1.52a0 Requires-Python >=3.9,<3.12; 0.1.53 Requires-Python >=3.9,<3.12; 0.1.54 Requires-Python >=3.9,<3.12; 0.1.55 Requires-Python >=3.9,<3.12; 0.1.55a0 Requires-Python >=3.9,<3.12; 0.1.55a1 Requires-Python >=3.9,<=3.12; 0.1.55a2 Requires-Python >=3.9,<=3.12.1; 0.1.55a3 Requires-Python >=3.9,<=3.12.1; 0.1.56 Requires-Python >=3.9,<3.12; 0.1.57 Requires-Python >=3.9,<3.12; 0.1.58 Requires-Python >=3.9,<3.12; 0.1.59 Requires-Python >=3.9,<3.12; 0.1.59a0 Requires-Python >=3.9,<3.12; 0.1.6 Requires-Python >=3.9,<3.12; 0.1.60 Requires-Python >=3.9,<3.12; 0.1.61 Requires-Python >=3.9,<3.12; 0.1.61a0 Requires-Python >=3.9,<3.12; 0.1.62 Requires-Python >=3.9,<3.12; 0.1.63 Requires-Python >=3.9,<3.12; 0.1.64 Requires-Python >=3.9,<3.12; 0.1.65 Requires-Python >=3.9,<3.12; 0.1.66 Requires-Python >=3.9,<3.12; 0.1.67 Requires-Python >=3.9,<3.12; 0.1.68 Requires-Python >=3.9,<3.12; 0.1.69 Requires-Python >=3.9,<3.12; 0.1.7 Requires-Python >=3.9,<3.12; 0.1.70 Requires-Python >=3.9,<3.12; 0.1.71 Requires-Python >=3.9,<3.12; 0.1.72 Requires-Python >=3.9,<3.12; 0.1.73 Requires-Python >=3.9,<3.12; 0.1.74 Requires-Python >=3.9,<3.12; 0.1.75 Requires-Python >=3.9,<3.12; 0.1.76 Requires-Python >=3.9,<3.12; 0.1.77 Requires-Python >=3.9,<3.12; 0.1.77a0 Requires-Python >=3.9,<3.12; 0.1.77a1 Requires-Python >=3.9,<3.12; 0.1.77a2 Requires-Python >=3.9,<3.12; 0.1.78 Requires-Python >=3.9,<3.12; 0.1.79 Requires-Python >=3.9,<3.12; 0.1.8 Requires-Python >=3.9,<3.12; 0.1.80 Requires-Python >=3.9,<3.12; 0.1.81 Requires-Python >=3.9,<3.12; 0.1.81a0 Requires-Python >=3.9,<3.12; 0.1.9 Requires-Python >=3.9,<3.12; 0.14.0rc0 Requires-Python >=3.10,<3.12; 0.5.12 Requires-Python >=3.7,<3.12; 0.5.13 Requires-Python >=3.7,<3.12; 0.6.2 Requires-Python >=2.7.0.0,<2.8.0.0,>=3.4.0.0,<4.0.0.0; 0.6.3 Requires-Python >=2.7.0.0,<2.8.0.0,>=3.4.0.0,<4.0.0.0; 4.0.0 Requires-Python >=3.7,<3.11
ERROR: Could not find a version that satisfies the requirement onnxruntime>=1.14.1 (from chromadb) (from versions: none)
ERROR: No matching distribution found for onnxruntime>=1.14.1
"

**Comment by pradipd25** [comment]
Does crewai Agents support conversational mode ?

**Comment by nmvega** [comment]
Hello Friends:

Is there any chance that this CrewAI GitHub [issue](https://github.com/crewAIInc/crewAI/issues/932) will be resolved. It's pretty important for **Observability** and troubleshooting. Thank you! =:)
